definitions/contested/json/CaseEventToFields/CaseEventToFields-generalOrder-nonprod.json

```diff
@@ -0,0 +1,216 @@
+[
```

Comment:
is there a reason this file is called general order when the rest are consent order?